client/src/components/JurisdictionAdmin/BatchRoundSteps/BatchRoundTallyEntry/BatchDetail.tsx

```diff
@@ -539,6 +546,12 @@ const BatchResultTallySheet: React.FC<IBatchResultTallySheetProps> = ({
         </BatchResultTallySheetTable>
 
         <BatchResultTallySheetButtonRow>
+          {batch.lastEditedBy && !isEditing && (
+            <LastEditedByText>
+              Last edited by <strong>{batch.lastEditedBy}</strong>
```

Comment:
I like the use of gray here to lower this in the visual hierarchy. Suggest using `Classes.TEXT_MUTED` for consistency with other gray text. The bold kind of cancels it out though, so I might remove the bold.

Also maybe adding a colon could make it a bit more readable (especially in the tally entry user case): e.g. `Last edited by: Shrek, Donkey`

Reply:
✅

Added an updated screenshot to the end of the PR description